### ISSUE TITLE
go-mysql replication performance question

### ISSUE THREAD

**Issue by guoyuanchao1202** [go-mysql-org/go-mysql#669]
I wonder how the performance of this part of go-mysql replication, do you guys have test data?
Since our project uses the go-mysql replication part, I now need to perform performance tests on go-mysql, so it would be great if I could get some of your test data

**Comment by lance6716** [comment]
This varies from your implementation and hardware, https://github.com/pingcap/tiflow/tree/master/dm can get 30k rows per seconds in replication

**Comment by guoyuanchao1202** [comment]
yeah, I mainly want to test the performance between BinligSyncr receiving the binlog, then parsing it, and finally sending it out through BinlogStreamer, so I'll do a test
